src/common/xspec-utils.xsl

```diff
@@ -274,6 +274,22 @@
 		</xsl:choose>
 	</xsl:function>
 
+	<!--
+		x:yes-no-synonym#1 plus default value
```

Comment:
Consider "x:yes-no-synonym#1 plus default value in case of empty sequence". Otherwise, it could be a default in case the input is an empty string or a string that doesn't match one of the expected values.

Reply:
Thanks for filling a missing point. Revised the comment as suggested and added a couple of tests. (8d20fb777f369f8b1bb87ba376b00e93f0b4fb2b)

Reply:
Looks great, including the extra tests.